modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SearchControls.js

```diff
@@ -27,8 +29,15 @@ const SearchControls = ({
 	searchInputName,
 	searchMobile,
 	searchValue,
-	setSearchMobile,
 }) => {
+	const searchInputRef = useRef(null);
+
+	useEffect(() => {
+		if (searchMobile && searchInputRef.current) {
+			searchInputRef.current.focus();
+		}
+	}, [searchMobile]);
```

Comment:
Do we need to set `null` as default for `searchInputRef`? It references an HTML element that we are unconditionally rendering. If no, check for `null` is also not needed.

Reply:
@markocikos thanks for reviewing this!  it's not necessary to set the ref to `null`. I have update the commit (https://github.com/liferay-frontend/liferay-portal/pull/3263/commits/3ccb762004d504ce9f14d77d8dfa260c9885a9ee) removing `searchInputRef` from the condition.